### DISCUSSION TITLE
Does order of subqueries matter?

### DISCUSSION THREAD

**Discussion by killJoy-03** [memgraph/memgraph#2556] [Q&A]
I am writing a READ query like this

match (a)-[b]-(c)-[d]-(e)
call{
sub query 1
}
call{
sub query 2
}
call{
sub query 3
}
return something

This give wrong result in subquery 3

but when I change the order from 1 ,2 ,3  -> 3,1,2 or 3,2,1
it gives right result

NOTE: In NEO4J alwas gives right result

**Comment by antejavor** [comment]
Hi @killJoy-03, so the Neo4j is now considered "correct" since the subqueries and  `CALL` are not well defined in the [opencypher](https://opencypher.org/). 

We have to double-check what the [GQL ](https://www.gqlstandards.org/what-is-a-gql-standard) standard have to say about this. 

Either way, the neo4j approach is probably the best one to follow at the moment, [here is the issue for this problem](https://github.com/memgraph/memgraph/issues/2560)

Thanks for communicating this.
